### PR TITLE
Fix some bugs

### PR DESCRIPTION
**Pull request by nightblade9**
- Game crashes on start-up
- A new game randomly showed an empty/black screen no matter how much you move
- Added ability to pin map generation to a specific seed

### CODE REVIEW

File: furusiya/main_interface.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class Game:
     game_msgs = []
     game_state = None
 
-    area_map: AreaMap = None
```
Comment (NegativeScript) [inline furusiya/main_interface.py]: Why did you remove this? It helps with auto completion :/

**Comment by NegativeScript** [comment]
LGTM. Good stuff.

File: furusiya/main_interface.py (hunk 1 above)
Comment (nightblade9) [reply furusiya/main_interface.py]: I removed it because it makes the game crash. I thought I replaced it with `area_map = None`.
